blackout/parserapp/templates/parserapp/delete/predelete.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+	{%load static%} 
+	<link href="{% static 'parserapp/output.css'%}" rel="stylesheet"> 
+</head>
+<body class="bg-gray-900 text-gray-100/75 pl-2">
```

Comment:
move to css

Reply:
It's class names. This names link to CSS